packages/client/src/components/RatingStars/RatingStars.jsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 import Rating from '@mui/material/Rating';
 import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
 
 export default function RatingStars(params) {
```

Comment:
params doesn't seem to be in use here, does it?

Reply:
Removed params as not used currently.